### DISCUSSION TITLE
Landing on issue from from app doesn't add title and tags

### DISCUSSION THREAD

**Discussion by bewuethr** [community/community#50983] [Mobile]
### Select Topic Area

Bug

### Body

A bit similar to #44032, but not generally about issue forms not being supported directly by the app.

When I leave the app to create an issue using the new Issue Form templates, I've noticed that the title isn't pre-filled, and the tags are missing.

For example, the template defined [here](https://github.com/bewuethr/progress/blob/main/.github/ISSUE_TEMPLATE/bridges.yml); when I open it from web, the URL reads

```none
https://github.com/bewuethr/progress/issues/new?assignees=&labels=workout&template=bridges.yml&title=Record+bridges
```

and everything is pre-filled as expected.

![image](https://user-images.githubusercontent.com/8521043/227689362-c551e7f0-99b8-4939-86d8-548f7b3d2071.png)

When I open it from the app, the URL is

```none
https://github.com/bewuethr/progress/issues/new?template=bridges.yml
```

amd indeed, the title isn't pre-filled and the label is missing:

![image](https://user-images.githubusercontent.com/8521043/227689451-9356e4f6-3194-436e-9c73-cf97969e314e.png)
![image](https://user-images.githubusercontent.com/8521043/227689456-a28e1ac8-b237-4361-afca-3bc12113ba73.png)

It seems that when exiting the app to the browser somehow drops a few query string parameters.

**Comment by iskae** [comment]
Hello @bewuethr 👋 

Thanks a bunch for reaching out, but unfortunately the `Mobile` category is only related to GitHub Mobile Apps and not the browser on mobile devices 😅 You may have a better chance if you reach out to our colleagues in [Issues](https://github.com/orgs/community/discussions/categories/issues?discussions_q=is%3Aopen+sort%3Adate_created+is%3Aunanswered+category%3AIssues) for this 🙇 

Cheers!

**Reply by bewuethr** [reply]
No, this is specifically when I open an issue from the mobile app: the app doesn't natively support the new issue forms yet, so it forwards to the browser, but in doing so cuts off query string parameters.

**Comment by blakeNaccarato** [comment]
It looks like this Discussion wasn't recategorized correctly. This does actually seem to be a Mobile app issue like OP intended, or at least a joint issue between the Mobile and Issues/Projects team.

Tapping an an issue template link in the **mobile app** tunnels out to the browser (that's fine), but the form isn't fully applied, e.g. labels are not applied.

This means that anyone creating issues from the mobile app are unintentionally bypassing the auto-labeling feature of the forms. **This bug doesn't occur in a mobile browser-only flow,** it **only happens when coming from the GitHub mobile app**.

**Reply by bewuethr** [reply]
Right, should be a mobile issue, let me check if I can change the category.

**Reply by bewuethr** [reply]
I could! Thanks for the pointer.

**Reply by blakeNaccarato** [reply]
Cool! Yeah, maybe this issue is also just in triage purgatory because maybe there's feature work ongoing to natively support the forms. For the time being it looks like a labeling bot will still be needed to label the edge cases for repositories with sufficient volume.

**Comment by TPS** [comment]
https://github.com/orgs/community/discussions/36533?

**Reply by bewuethr** [reply]
I think the issue from that specific discussion has been fixed: the correct template is opened, but all other query string parameters are ignored.
